### ISSUE TITLE
Open Source License!

### ISSUE THREAD

**Issue by shiffman** [mikeboydvideo/MikeBoydGame#2]
Amazing work, wow! I do not want to create extra work for you or be presumptuous or claim to really know what I'm talking about. . .but since I ❤️❤️❤️ the world of open source this might be a nice moment to learn about open source practices and explicitly include a software license that lets people know what they can / cannot do with your code. I might suggest the MIT License for the source code and a "creative commons non-commercial" for your designs/sounds. This would allow anyone to repurpose the code (even make a similar game that they could sell). But for the design assets you might prefer someone not be able to re-use and sell them (like make a t-shirt with the images without your permission)?

This is a really helpful website:
https://choosealicense.com/

And more about Creative Commons:
https://creativecommons.org/choose/

Happy to help walk you through / update your GitHub repo!

🚂 🌈 💫❤️

**Comment by tterrag1098** [comment]
CreativeCommons licenses shouldn't be used for code. See: https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software

**Comment by d-siganos** [comment]
@mikeboydvideo I suggest you add the MIT license

**Comment by shiffman** [comment]
@tterrag1098 yes, good point! I was suggestions the CC licenses for [the media assets](https://github.com/mikeboydvideo/MikeBoydGame/tree/gh-pages/Wine_22/assets) which can be considered separate from the code!
